packages/headless/src/features/product-listing/product-listing-analytics.actions.ts

```diff
@@ -0,0 +1,32 @@
+import {Result} from '../..';
+import {ProductListingAnalyticsProvider} from '../../api/analytics/product-listing-analytics';
+import {
+  AnalyticsType,
+  ClickAction,
+  documentIdentifier,
+  makeAnalyticsAction,
+  partialDocumentInformation,
+  ProductListingAction,
+  validateResultPayload,
+} from '../analytics/analytics-utils';
+
+export const logProductListing = (): ProductListingAction =>
+  makeAnalyticsAction(
+    'analytics/productListing/load',
+    AnalyticsType.Search,
+    (client) => client.makeInterfaceLoad(),
+    (getState) => new ProductListingAnalyticsProvider(getState)
+  );
+
+export const logProductListingOpen = (result: Result): ClickAction =>
+  makeAnalyticsAction(
+    'analytics/productListing/open',
+    AnalyticsType.Click,
+    (client, state) => {
+      validateResultPayload(result);
+      return client.makeDocumentOpen(
```

Comment:
Same thing here, should I add a `makeListingOpen` ?

Reply:
I think reusing `makeDocumentOpen` is correct, since the click event ultimately looks the same, whether it is from a search page or PLP.

Reply:
+1